frontend/src/components/Charts/helpers/OrderTooltip/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useContext } from 'react';
```

Comment:
Previously there were some changes on the tooltip to make it work for Federation and I just include it here